packages/experiments/src/components/Button/BaseButton.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { Button } from './Button';
+import { IButtonComponent, IButtonStylesReturnType, IButtonTokenReturnType } from './Button.types';
+import { ButtonVariantsType } from './ButtonVariants.types';
+
+const baseTokens: IButtonComponent['tokens'] = {
+  backgroundColor: undefined,
+  backgroundColorHovered: undefined,
+  backgroundColorPressed: undefined,
+  borderColor: undefined,
+  borderColorHovered: undefined,
+  borderColorPressed: undefined,
+  borderRadius: undefined,
+  borderWidth: 2,
+  color: undefined,
+  colorHovered: undefined,
+  colorPressed: undefined,
+  contentPadding: 0,
+  cursor: 'default',
+  minHeight: undefined,
+  minWidth: undefined,
```

Comment:
Hmm, token erasure? This is a bit concerning if this will have to change over time if button styling is changed. Would this be easier or harder if there were no base tokens present?

Reply:
Well, I think this is a very specific case with `BaseButton` because of how it emulates an `unstyled` `Button`. Maybe it would be easier if no base tokens were present but I do think that `Button` as is should be styled like `DefaultButton` instead of like `BaseButton`.

Reply:
Does it make sense to have BaseButton be a more atomic version of Button that Button builds off of?

Reply:
Maybe, they would be different just in styling though. Any thoughts about that?

I feel like this is yet another use case of `extendComponent`.

Reply:
Yeah, the thing I would like to do is have things layered in a way where one doesn't erase the other because that will be harder to maintain.